composer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "SalesOrders",
```

Comment:
I think for packagist the name needs to be `vendasta/sales-orders`

Reply:
Ah yes, I ran into that for sure.